content/source/docs/cloud/registry/using.html.md

```diff
@@ -55,6 +55,16 @@ Click a module's "Details" button to view its details page. Use the "Versions" d
 
 ![Terraform Cloud screenshot: a module details page](./images/publish-module-details.png)
 
+### Viewing Nested Modules
+If a module contains nested modules following the [standard module structure](/docs/modules/structure.html), then a "Submodules" dropdown will appear below the module source information. 
+
+![Terraform Cloud screenshot: a module submodules button](./images/publish-submodules-dropdown.png)
```

Comment:
I thought it was weird that these images would have `publish-` in the filename, since this is the "using modules" page rather than the "publishing modules" page, and then I noticed that the last image file referenced above in the existing content has `publish-`! 

_90% chance that was my fault_ — I probably wrote this page second, and just re-used an existing screenshot that was generic enough to use in either context. Sorry about that! 

Still, it's probably worth renaming these two files to start with `using-` instead, just so that _most_ of these images stay clearly associated with a particular page.

Reply:
ah, good catch. I have updated the file names. Hopefully the publish page isn't confusing because I updated the publish-modules-details file to match the current prod UI. The images above it still say `beta` on the publish page.